app/models/users.js

```diff
@@ -1,18 +1,49 @@
-'use strict';
-
-var mongoose = require('mongoose');
-var Schema = mongoose.Schema;
-
-var User = new Schema({
-	name : String,
-	username: String,
-	password: String,
-	admin: Boolean,
-	fbid: String,
-	email: String,
-	imagepath: String,
-	role: String,
-	order: Number
-});
-
-module.exports = mongoose.model('User', User);
+'use strict';
+
+var mongoose = require("mongoose"),
+    Schema = mongoose.Schema,
+    bcrypt = require("bcrypt"),
+    SALT_WORK_FACTOR = 10;

```

Comment:
What is its function exactly?

Reply:
Password hashing using bcrypt. This is the number of rounds it uses to hash passwords I believe.